src/optsummary.jl

```diff
@@ -31,63 +31,40 @@ The latter four fields are MixedModels functionality and not related directly to
     the future to use a different subtype of `AbstractVector` (e.g., `StaticArrays.SVector`)
     for each snapshot without being considered a breaking change.
 """
-mutable struct OptSummary{T<:AbstractFloat}
+Base.@kwdef mutable struct OptSummary{T<:AbstractFloat}
```

Comment:
I hope that this will make some things easier to maintain

Reply:
Good idea.